src/test/groovy/com/gooddata/md/visualization/VisualizationObjectTest.groovy

```diff
@@ -201,6 +207,7 @@ class VisualizationObjectTest extends Specification {
     }
 
     def "test serializable"() {
+        setup:
```

Comment:
that `roundtrip` is actually the test, pls reorganize to `when - then` manner

Reply:
fixed